source/blog/2018-01-22-CICD-workflow-with-zuul.html.md

```diff
@@ -0,0 +1,407 @@
+---
+title: Overview of a CI/CD workflow with Zuul
+date: 2018-01-22
+author: tristanC
+tags: software-factory, zuul, ci, cd
+---
+
+The upcoming version of [Zuul](https://docs.openstack.org/infra/zuul/)
+has many new features that allow one to create powerful continuous
+integration and continuous deployment pipelines.
+
+This article presents some mechanisms to create such pipelines using
+a rpm package distribution development workflow as a practical example.
+
+To experiment with the configuration sample,
```

Comment:
This paragraph could have been placed at the end. Like "experiment Zuul by yourself"

Reply:
Done